types/braintree/index.d.ts

```diff
@@ -0,0 +1,1749 @@
+// Type definitions for braintree 2.16
+// Project: https://github.com/braintree/braintree_node
+// Definitions by: Sam Rubin <https://github.com/smrubin>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+export = braintree;
+export as namespace braintree;
+
+declare namespace braintree {
+    /**
+     * Braintree Config and Client
+     */
+
+    export enum Environment {
```

Comment:
The environment from config's values are actually 'Production' and 'Sandbox' right? If so, then you should map this enum to the strings a la

```
export enum Environment {
   Production = 'Production',
   Sandbox = 'Sandbox'
}
```

I think having Environment be a union type would be fine too.
```
type Environment = 'Sandbox' | 'Production';
```

Reply:
Agreed. I like the union type, will implement.